org.eclipse.scout.rt.server.commons/src/main/java/org/eclipse/scout/rt/server/commons/GlobalTrustManager.java

```diff
@@ -224,7 +228,8 @@ protected List<X509Certificate> getTrustedCertificatesInRemoteFiles() {
       return remoteFilesToCertificates(certRemoteFiles);
     }
     catch (RuntimeException e) {
-      LOG.error("Could not access folder '{}' to import trusted certificates.", PATH_CERTS, e);
+      RuntimeException e0 = LOG.isDebugEnabled() ? e : null;
```

Comment:
Is this "ok" to happen? If remote file service is available it should return a value?

Reply:
Yes, but remote file server may not be configured properly or there may be not PATH_CERTS path on remote file system.